packages/plugin-ext/src/plugin/scm.ts

```diff
@@ -669,7 +692,7 @@ class SourceControlImpl implements theia.SourceControl {
 
             const { handle, id, label, features } = group;
             groups.push({ handle, id, label, features });
-
+            group.setPlugin(this.plugin);
```

Comment:
I'm not sure about each scm-resource-group needing to set the `plugin`.

Reply:
You are right.
I'll check that to see which group to apply it to.